lib/cdo/shared_constants/pd/facilitator1920_application_constants.rb

```diff
@@ -204,5 +204,112 @@ def self.clean_multiline(string)
+      currently_involved_in_cs_education: 'How are you currently involved in CS education?',
+      grades_taught: 'If you do have classroom teaching experience, what grade levels have you taught? Check all that apply.',
+      experience_teaching_this_course: 'Do you have experience teaching the full {{CS Program}} curriculum to students?',
+      plan_on_teaching: 'Do you plan on teaching this course in the 2019-20 school year?',
+      csd_csp_completed_pd: "Have you participated as a teacher in Code.org's full Professional Learning Program for {{CS Program}}?",
+      csf_previous_workshop: 'Have you attended a Code.org CS Fundamentals workshop?',
+      facilitator_availability: 'When do you anticipate being able to facilitate? Note that depending on the program, workshops may be hosted on Saturdays or Sundays.',
+      why_should_all_have_access: clean_multiline(
+        "Code.org's Professional Learning Programs are open to all teachers, regardless of their experience with CS education.
+        Why do you think Code.org believes that all teachers should have access to the opportunity to teach CS?"
+      ),
+      skills_areas_to_improve: clean_multiline(
+        "Please describe a workshop you've led (or a lesson you've taught, if you haven't facilitated a workshop). Include a brief description of the workshop/lesson
+        topic and audience (one or two sentences). Then describe two strengths you demonstrated, as well as two facilitation skills you would like to improve.",
+      ),
+      inquiry_based_learning: clean_multiline(
+        "Code.org Professional Learning experiences incorporate inquiry-based learning into the workshops. Please briefly define  inquiry-based
+        learning as you understand it (one or two sentences). Then, if you have led an inquiry-based activity for students, provide a concrete
+        example of an inquiry-based lesson or activity you led. If you have not led an inquiry-based lesson, please write 'N/A.'",
+      ),
+      why_interested: 'Why do you want to become a Code.org facilitator? Please describe what you hope to learn and the impact you hope to make.',
+      anything_else: 'Is there anything else you would like us to know? You can provide a link to your resume, LinkedIn profile, website, or summarize your relevant past experience.',
+      how_heard: 'How did you hear about this opportunity?',
+      question_1: "Interview #{INTERVIEW_QUESTIONS[:question_1]}",
+      question_2: "Interview #{INTERVIEW_QUESTIONS[:question_2]}",
+      question_3: "Interview #{INTERVIEW_QUESTIONS[:question_3]}",
+      question_4: "Interview #{INTERVIEW_QUESTIONS[:question_4]}",
+      question_5: "Interview #{INTERVIEW_QUESTIONS[:question_5]}",
+      question_6: "Interview #{INTERVIEW_QUESTIONS[:question_6]}",
+      question_7: "Interview #{INTERVIEW_QUESTIONS[:question_7]}"
```

Comment:
Since our PD is currently US-only (except specific arrangements with a few international partners), none of it is translated. These strings are similar to the application questions, which are stored in a constants file, but I shortened some of them for brevity in the CSV header (although they are still long).